docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -1,34 +1,53 @@
-==========================
-File system access control
-==========================
+=========================
+File-based access control
+=========================
 
-This access control plugin allows you to specify authorization rules in a JSON file.
+To secure access to data in your cluster, you can implement file-based access
+control where access to data and operations is defined by rules declared in
+manually-configured JSON files.
+
+There are two types of file-based access control:
+
+* **System-level access control** uses the access control plugin with a single
+  JSON file that specifies authorization rules for the whole cluster.
+* **Catalog-level access control** uses individual JSON files for each catalog
+  for granular control over the data in that catalog, including column-level
+  authorization.
```

Comment:
How do these two types of access control interact? How is precedence handled if there's a catalog file and system file that have contradicting rules?

Reply:
Chatted with @electrum , the way this works is that `deny` rules always take precedence over `allow`. So if there is a rule in one file that conflicts with another, the operation will be denied according to either rule. As a result, there's no favoring of system vs catalog. Will add to this PR shortly.